### DISCUSSION TITLE
Bubbles cover digits

### DISCUSSION THREAD

**Discussion by Bouni** [jomjol/AI-on-the-edge-device#3399] [General]
Hi,

I wonder if anybody else has the same issue as I'm facing right now.

In my watermeter is a big bubble of air that covers the digits.

It is not always in the same place and sometimes it's even gone completely.

![Screenshot_2024-11-21-19-43-07-57_3aea4af51f236e4932235fdada7d1643.jpg](https://github.com/user-attachments/assets/36411e8a-5c75-4ac0-9acc-e56376785a9d)

Does anybody have a solution for this kind of issue? 

I thought maybe I can close the main valve and then loosen the meter a bit and fix it again in a slightly angled position so that the bubble is forced to the edge .

No idea if that's feasible to be honest...

I'm thankful for every tip that helps solving this issue

**Comment by friedpa** [comment]
Turn the water meter a little bit and the bubble will disappear to the side. You have still to improve the picture a lot. See all the tips in the Wiki how to improve the picture taken by the ESP.
My sample below as a guideline, which works without any hassle: 

![image](https://github.com/user-attachments/assets/c5ae55c8-d869-4cc6-b39e-bc9ca458817c)

**Comment by Bouni** [comment]
Ok, thank you for your inputs.
I'll try and give feedback if it works 👍
